dwave/plugins/torch/utils.py

```diff
@@ -1,63 +1,19 @@
-# Copyright 2025 D-Wave
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
```

Comment:
Is this license removal intentional?

Reply:
Nope, should be added back. I think it got removed because the whole file was (at first) removed and then never got added back when the file was resurrected.